_posts/01-06-01-Linux-Setup.md

```diff
@@ -25,7 +29,7 @@ For Ubuntu distributions, the [PPA by Ondřej Surý][Ondrej Sury PPA] provides s
 
 This will ensure that your system can access and install the latest PHP packages available in the PPA.
 
-#### Debian-based distributions
+### Debian and other Debian-based distributions
```

Comment:
Would it make more sense just to call this section `Debian-based distributions`? `Debian and Debian-based` feels a little redundant, especially since above on line 10 its just `Debian-based`.

Reply:
Perhaps so, yes. I've added another commit in the PR now which fixes this a bit.